yarn.lock

```diff
@@ -6025,6 +6025,11 @@ axe-core@3.5.0:
   resolved "https://registry.yarnpkg.com/axe-core/-/axe-core-3.5.0.tgz#161b17fc6ce07a5470ff179e0703957c422b2220"
   integrity sha512-wJqOIreoFiGjvZ1UZvGLAUs8H3QQ3cS833+6ctFcCdr/xFd5oB66mmwGWnwQlBXFFaefRt+KR+m2dY9em2RxVg==
 
+axe-core@^3.0.3:
```

Comment:
Probably not important in this case, but it looks like the addition of this extra axe-core version is unnecessary (maybe a leftover from some previous version of the change). So you could revert this diff and the one above.

Reply:
this was generated by `yarn` any manual change will make all contributors quite a hard time as after running `yarn` they would have modified changes.

Reply:
> any manual change will make all contributors quite a hard time as after running `yarn` they would have modified changes.

It won't as long as:
1. the combined version spec satisfies everything that's being requested (it would in this case--the full set of requested versions is the same before and after, so this actually shouldn't have changed at all; it probably only did as a leftover from some other intermediate changes)
1. you run `yarn` yourself after any manual version tweaks, and include any further changes in the commit

This is something I've ended up doing a good bit of manually while we're still working on improving tooling in the area. It's more impactful when the initial duplicate is a package like `@babel/core` which has a whole big tree of dependencies.